docs/sharing.md

```diff
@@ -136,6 +140,20 @@ Content-Type: application/vnd.api+json
           "status": "mail-not-sent",
           "name": "Bob",
           "email": "bob@example.net"
+        },
+        {
+          "status": "mail-not-sent",
+          "name": "Gaby",
+          "email": "gaby@example.net",
+          "only_in_groups": true,
+          "groups": [0]
```

Comment:
I'm more comfortable with this version where the groups are referenced in the members 👍 

As a personal question, I preferred to use the id rather than the position in an array, but I can't think of any rational reason why not

Reply:
Using indexes means their order should never change or members should be updated if the groups order changes.

Unless there's a very good reason to use indexes, I thinks we should use ids here to avoid issues.

Reply:
The order of the groups will never change